inspirehep/modules/workflows/workflows/hep_ingestion.py

```diff
@@ -125,6 +130,11 @@ class mysource(HEPIngestion):
             with_author_keywords=True,
         ),
         filter_core_keywords,
+        guess_categories,
+        IF(is_experimental_paper, [
+            guess_experiments,
+        ]),
+        guess_keywords,
```

Comment:
Yeah. You make some good points. I will look into this shortly on organizing these workflows altogether. For this PR however, we can leave it I think.